docs/Configuration.md

```diff
@@ -748,6 +748,12 @@ class MyCustomReporter {
 
 For the full list of methods and argument types see `Reporter` interface in [packages/jest-reporters/src/types.ts](https://github.com/facebook/jest/blob/main/packages/jest-reporters/src/types.ts)
 
+### `reportPhasingResults` [boolean]
```

Comment:
instead of a separate option, should we pass this as a reporter option?

Reply:
I'm worried that reporters will not handle the new "assertions" (which aren't assertions); there's no safe default for that option.

We could make it a reporter option, and have a breaking change for all reporters, which seems expensive.